src/tracebox/tracebox.cc

```diff
@@ -52,6 +52,8 @@ using namespace std;
 static bool skip_suid_check = false;
 
 static int hops_max = 64;
+static int hops_min = 0;
```

Comment:
Make it start directly at 1. Setting it as uint8_t should not hurt as well as it would remove the need to check for negative values.